app/src/main/java/com/dankideacentral/dic/activities/TweetFeedActivity.java

```diff
@@ -172,100 +164,29 @@ public boolean handleMessage(Message msg) {
                 String tweetUserName = tweet.getUser().getName();
                 long tweetUserId = tweet.getUser().getId();
                 if(friends.contains(tweetUserId)) {
-                    sendNotification("Your friend " + tweetUserName + " tweeted near you!");
+                    notificationHandler.sendNotification("Your friend " + tweetUserName + " tweeted near you!",
+                            notificationId++);
```

Comment:
Do you think the TweetFeedActivity should be concerned with notification ID's any more? Or should the NotificationHandler deal with them?